frontend/src/App.tsx

```diff
@@ -1,29 +1,29 @@
-
 import React from 'react';
 import './App.css';
 import {useAppDispatch} from "./app/hooks";
 import {getEmployees} from "./slicer/employeeSlice";
 import {loginFromStorage} from "./slicer/authSlice";
 
 //components
-import { CssBaseline, Toolbar } from '@mui/material';
+import {CssBaseline, Toolbar} from '@mui/material';
 import Header from './components/header/Header';
 import MainView from './components/main-view/MainView';
-
+import Drawer from "./components/drawer/Drawer";
```

Comment:
unused component?

Reply:
one of four. I should have checked the tests before committing the pr....